### PR TITLE
Fixes #15240: Move /opt/rudder/etc/ssl/nodeslist.cert to /var/rudder/lib/ssl - techniques

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/15240

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: techniques/system/distributePolicy/1.0/apache-acl.cf [hunk 1]
```diff
@@ -25,7 +25,7 @@ bundle agent apache_acl
       "ssl_ca_size"        string => filestat("${this.promise_dirname}/${ssl_ca_file}", "size");
 
     pass1::
-      "src_ca_file" string => "${this.promise_dirname}/${ssl_ca_file}",
+      "src_ca_file" string => "${this.promise_dirname}/nodescerts.pem",
                 ifvarclass => "!empty_ssl_ca";
```
Comment (peckpeck) [inline techniques/system/distributePolicy/1.0/apache-acl.cf]: shouldn't you change the ssl_ca_file variable instead ?

Comment (peckpeck) [reply techniques/system/distributePolicy/1.0/apache-acl.cf]: still waiting for an answer

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**